cgi-bin/DW/User/Rename.pm

```diff
@@ -229,10 +229,14 @@ sub swap_usernames {
     my $admin = LJ::isu( $opts{user} ) ? $opts{user} : $u1;
     my @tokens = @{ $opts{tokens} || [] };
 
+    # tokens can be owned by $admin (remote user),
+    # $u1 (authas user), or $u2 (intended target)
+    my $check_uids = { $admin->id => 1, $u1->id => 1, $u2->id => 1 };
```

Comment:
Style-wise, I'd prefer hashes over hashrefs. Less syntactic sugar involved in the usage, and morally, not dereferencing a pointer should be faster. (Although quite irrelevant in this usage.)